### ISSUE TITLE
Improve support for Elementary OS

### ISSUE THREAD

**Issue by kamilprusko** [codito/gnome-pomodoro#276]
Tested latest gnome-pomodoro with Loki... it's not usable

- the app icon is blurry (when installed from source)
- notifications stay on the screen, should be transient or lower priority?
- notification messages are not updated, not much of an issue if they landed in the notifications menu
- preferences dialog does not accept new styles due to an old version of Gtk+ 3.18 
- indicator menu show toggles for actions like "Quit" and "Preferences"
- indicator menu items order is messed up
- screen notification resizes during open - the fade-in animation is choppy
- changing durations shows a slew of new notifications
- missing icon theme "elementary" for the indicator
- dark-theme plugin crashing due to missing symbol ```gtk_settings_reset_property```, again due to old version of Gtk

I'm not sure if there are any gnome-pomodoro users on Elementary. Gtk issues could be ignored assuming improvements would target next version of Elementary.

**Comment by dlbnco** [comment]
Yep, installed Juno yesterday and the Pomodoro indicator is not being shown anymore.

**Comment by kamilprusko** [comment]
I think the desktop integration on Elementary OS is good enough...

It's better than on some desktops like Cinnamon which lacks idle/activity detection. It's missing the icon in the system tray / wingpanel, but the app now has a compact mode to fill this gap. The [download rates](https://github.com/focustimerhq/gnome-shell-extension-focus-timer/releases/tag/3) for GNOME Shell extension are quite low, so it's not encouraging for making dedicated integrations that are not bundled with the app.

I'm sure it wouldn't be hard to use AI to rewrite the indicator from GNOME Shell extension into GTK+ / wingpanel API; but it would still require extra maintenance, and the target user base would be much smaller than that of GNOME Shell extension...
